Vireo_VS/VireoCommandLine.vcxproj

```diff
@@ -96,7 +96,7 @@
     <UseDebugLibraries>false</UseDebugLibraries>
     <PlatformToolset>v140</PlatformToolset>
     <WholeProgramOptimization>true</WholeProgramOptimization>
-    <CharacterSet>Unicode</CharacterSet>
+    <CharacterSet>MultiByte</CharacterSet>
```

Comment:
Looks like I spoke too soon. Looks like there are some file system code that is already using mult-byte/ascii dependent Win32 APIs. If we want to go to Unicode, we have to fix those. I don't see a reason to put any effort to do that now. I have set the character set to "Not Set" in the VC project now. Again, this change will simply let us build both release and debug with similar configurations.